### PR TITLE
Live plotting refresh rate selector

### PR DESCRIPTION
**Pull request by DavidSarlle**
This PR implements a selector for customizing the refresh interval time in every plot of qq-live. 
Auto = time neeeded for each plotting function to retreive the data a create the figure
No refresh = stop refreshing graphs

The setting is independent for each action/graph showed in a report.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibocal/web/app.py [hunk 1]
```diff
@@ -58,7 +105,21 @@ def get_graph(n, current_figure, url):
...
+        et = time.time()
+        if value == 0:
+            refresh_rate = (et - st) + 6
+        else:
+            refresh_rate = value
 
-        return getattr(plots, method)(folder, routine, qubit, format)
+        return (
+            figure,
+            [html.Span(f"Last update: {datetime.datetime.now()}")],
```
Comment (andrea-pasquale) [inline src/qibocal/web/app.py:117]: image
I think that here it should be sufficient to show the timestamp up to the seconds.

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @DavidSarlle.
Looks good to me. You can find below a small comment.

File: src/qibocal/web/app.py (hunk 1 above)
Comment (DavidSarlle) [reply src/qibocal/web/app.py]: Thanks @andrea-pasquale for your comment. Implemented. If there are no more comments the PR is ready to be merged into main from my side.

**Review by andrea-pasquale — APPROVED** [review]
Thanks @DavidSarlle.

**Comment by maxhant** [comment]
Hi,
I tried merging your branch to mine to have this functunalities. I am not sure it will be reprocible but I get this error:
```
-------------------------------------------------------------------------------
qq 8 <module>
sys.exit(command())

core.py 1130 __call__
return self.main(*args, **kwargs)

core.py 1055 main
rv = self.invoke(ctx)

core.py 1404 invoke
return ctx.invoke(self.callback, **ctx.params)

core.py 760 invoke
return __callback(*args, **kwargs)

_base.py 61 command
builder.dump_report()

builders.py 199 dump_report
create_report(self.folder)

report.py 21 create_report
html = template.render(

environment.py 1301 render
self.environment.handle_exception()

environment.py 936 handle_exception
raise rewrite_traceback_stack(source=source)

template.html 183 top-level template code
{{ report.get_figure(routine, method, qubit) }}

builders.py 251 get_figure
figure.write_html(temp.name, include_plotlyjs=False, full_html=False)

AttributeError:
'list' object has no attribute 'write_html'
```
I think that those few lines need to be changed:
https://github.com/qiboteam/qibocal/blob/7c7c86fa124c06cd7ca347d5d3c91d03016fe307/src/qibocal/cli/builders.py#L248-L252

**Comment by DavidSarlle** [comment]
Thanks @maxhant for reporting the error. I have modificated the get_figure method to be able to manage multiple figures returned from the plotting functions. This modification should fixes the error.

Please @maxhant and maybe other reviewer (@andrea-pasquale @aorgazf @scarrazza ) take a look.

From my side it is working and  this PR should be now ready to be merged on main.

**Comment by maxhant** [comment]
You just need to merge `main` because some routines are currently crashing like `calibrate_qubit_state`, but after merging things are working fine. 
The only problem that I am getting is when using interactive zooming. When I try to zoom, the selection area is much larger than the one where my mouse is. I am trying to represent it below, where my mouse tick is the red dot. Is it doing the same to you @DavidSarlle? 

image

**Comment by DavidSarlle** [comment]
Thanks @maxhant for the review. 

Regarding the zoom problem, no, I have no problems. I can zoom the area and it results in a correct zoom in of the are selected with my mouse. check the images below:

1. Selecting area
image

2. Zoom in of selected area:
image

**Review by maxhant — APPROVED** [review]

**Pull request merged**